server/modules/serving.js

```diff
@@ -37,6 +35,32 @@ function serveFile(path) {
   };
 }
 
+const moduleRoutes = new Map;
+// Register a route. We need to refcount because we could have two copies of
+// the same module playing.
```

Comment:
Would per-instantiation-id paths be better?
I suspect it's overkill given the content served by modules is so static today, but thought I'd mention it.

Reply:
Noted.

I think this would come up if, for example, you moved the files associated with a module, and then told the wall to switch to showing it again under the new path. I could imagine that happening, but it's not worth writing this code for now.